pkg/registration/register.go

```diff
@@ -25,7 +26,7 @@ func RegisterUser(ctx context.Context, username string, password string, email s
 		return errors.New("username can only contain letters, digits, period, hyphen or underscore")
 	}
 	// Should we have other unacceptable usernames?
-	if strings.ToLower(username) == "anonymous" {
+	if strings.ToLower(username) == "anonymous" || username == utilities.CensoredUsername {
```

Comment:
need to disallow registering with alternate capitalization
```suggestion
	if strings.EqualFold(username, "anonymous") || strings.EqualFold(username, utilities.CensoredUsername) {
```